google-cloud-spanner/src/main/java/com/google/cloud/spanner/v1/stub/GrpcSpannerStub.java

```diff
@@ -247,6 +247,11 @@ public static final GrpcSpannerStub create(ClientContext clientContext) throws I
     return new GrpcSpannerStub(SpannerStubSettings.newBuilder().build(), clientContext);
   }
 
+  public static final GrpcSpannerStub create(
```

Comment:
This is a generated file, so you can't manually update it. (That is: you can update it, but the next round of code generation will overwrite your changes)